File: iOS/Anmat Vademecum/Formula.m
//
//  Formula.m
//  VNM
//
//  Created by mag on 4/11/15.
//  Copyright (c) 2015 Think Up Studios. All rights reserved.
//

#import "Formula.h"

@implementation Formula {
    NSMutableArray *components;
}

- (id) init {
    components = [[NSMutableArray alloc] init];
    
    return self;
}

- (NSArray *)getComponents {
    return components;
}

- (void) addComponent:(Component *)component {
    [components addObject:component];
}

@end

File: Android/vademecum/src/main/java/com/thinkupstudios/anmat/vademecum/bo/Formula.java
package com.thinkupstudios.anmat.vademecum.bo;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * Created by dcamarro on 11/04/2015.
 */
public class Formula implements Serializable{

    private List<Component> components;

    public Formula()
    {
        this.components = new ArrayList<>();
    }

    public List<Component> getComponents() {
        return this.components;
    }

    public void addComponent(Component component) {
        this.components.add(component);
    }
}
